src/app/update-profile-page/update-profile/update-profile.component.ts

```diff
@@ -170,7 +169,7 @@ export class UpdateProfileComponent implements OnInit, OnChanges {
       this.profileUpdates.usernameUpdate /*NewUsername*/,
       this.profileUpdates.descriptionUpdate /*NewDescription*/,
       this.profileUpdates.profilePicUpdate /*NewProfilePic*/,
-      this.founderRewardInput * 100 /*NewCreatorBasisPoints*/,
+      this.founderRewardInput * 1000 /*NewCreatorBasisPoints*/,
```

Comment:
this is in basis points, so multiplying by 1000 will result in an error.  You can't have a founder's reward > 100%.

Reply:
Right! I knew I should have reviewed that more closely. It was a last minute quick addition. Sry!!! The text should be updated at least.